### PR TITLE
Clean up some weird command strings

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @kinnison

### CODE REVIEW

File: src/librustdoc/lib.rs [hunk 1]
```diff
@@ -248,8 +247,8 @@ fn opts() -> Vec<RustcOptGroup> {
                 "e",
                 "extend-css",
                 "To add some CSS rules with a given file to generate doc with your \
-                      own theme. However, your theme might break if the rustdoc's generated HTML \
-                      changes, so be careful!",
+                        own theme. However, your theme might break if the rustdoc's generated HTML \
```
Comment (hbina) [inline src/librustdoc/lib.rs:250]: Wouldn't this cause the text to have double space?

**Comment by kinnison** [comment]
A couple of these look like nice cleanups, but some of the reindentation seems arbitrary, it's surely not rustfmt so why the extra indentation?

File: src/librustdoc/lib.rs (hunk 1 above)
Comment (Lonami) [reply src/librustdoc/lib.rs:250]: As far as I know the `\` above followed by a newline will "eat" all whitespace in the next line(s) until a non-space character, so indenting it more is not an issue.

**Comment by GuillaumeGomez** [comment]
I tried to pick the "most common" one. But if you have a preference, I'm completely open to it.

**Comment by kinnison** [comment]
I don't mind, it just felt odd to see them, but if you're balancing them out with other multiline strings elsewhere then that's fine.

@bors r+ rollup

**Pull request merged**